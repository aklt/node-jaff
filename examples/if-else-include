#if false
branch1
#include if-else-false
#elif false
branch2
#include if-else-nest-false
#else
branch3
#include if-else-true
#end
