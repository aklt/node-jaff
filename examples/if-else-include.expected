branch3
before
branch 1
after
